scripts/runtest.sh

```diff
@@ -99,11 +99,11 @@ function short_test {
     fi
     if [ -n "$VERBOSE" ]
     then
-        echo "VCD_SHORT_TEST=1 go test -race -tags 'functional $MORE_TAGS' -v -timeout 5m"
+        echo "VCD_SHORT_TEST=1 go test -race -tags '$MORE_TAGS' -v -timeout 5m"
```

Comment:
`functional` tag was default until TM, but this tag is now moved upstream to the function caller and it should be included within `$MORE_TAGS` in the caller. This allows for isolating it